Frontend/src/components/Input/TextInput.svelte

```diff
@@ -72,4 +80,11 @@
     resize: none;
     overflow: hidden;
   }
+  .button-tight {
```

Comment:
Since this is the same code as in DateInput.svelte we should make a new component for this tight button and use it in DateInput and TextInput to prevent duplicate code.

Reply:
done!